README.md

```diff
@@ -148,6 +148,9 @@ $ cd liquidctl
 $ pytest  # optional step
 $ python -m liquidctl.cli <args>...
 # pip install .
+# sudo cp liquidctl-completion.sh /etc/bash_completion.d/	# to install bash completion script
+# sudo cp liquidctl.8 /usr/local/share/man/man8/
+# sudo mandb
```

Comment:
I'm not sure about suggesting this for all manual installs.  These files wouldn't be tracked, would eventually become trash, and could also conflict/interact badly with a later install of a distro package of liquidctl.

Can you place these into a separate code block, suggesting that they are optional steps?

And since we're already taking care of `liquidctl.8`, the same could be done with the udev rules.

Reply:
yup I'll move this to an optional step.